themes/mlibrary_new/common/header.php

```diff
@@ -58,8 +58,8 @@
         </div>
     </div>
 
-    <?php if ((get_theme_option('display_header') !== '0')): ?>
-    <header id="banner" class="<?php echo get_theme_option('header_flow'); ?> page-header" style="background-size:cover;background-image:url('<?php 
+<?php if (((get_theme_option('display_header') !== '0')) and (end(explode('/',$_SERVER['REQUEST_URI']))!='about')) : ?>   
+     <header id="banner" class="<?php echo get_theme_option('header_flow'); ?> page-header" style="background-size:cover;background-image:url('<?php 
 		if ((get_theme_option('Header Background Image') === null)){
```

Comment:
This isn't in this patch, but this logic should be in bs_header_bg().

Reply:
Are you talking about the if condition or code starting from <header> tag?

Reply:
@bridgetburke is that can be moved to bs_header_bg()

Reply:
Lines 63-65 should be an else clause in `bs_header_bg()`.

This wasn't part of the changes here, so feel free to consider it out of scope of this review.

Reply:
@nancymou , if you want me to fix this, please create and issue and assign me after your merge this. Thanks.